frontend/packages/kubevirt-plugin/src/components/connected-vm-console/vm-console-empty-state.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import {
+  Title,
+  EmptyState,
+  EmptyStateIcon,
+  TextContent,
+  EmptyStateBody,
+  TextList,
+  TextListItem,
+  TextListVariants,
+  TextListItemVariants,
+} from '@patternfly/react-core';
+import { PlugIcon, ErrorCircleOIcon } from '@patternfly/react-icons';
+import './vm-console-empty-state.scss';
+
+export interface ConsoleEmptyStateProps {
+  isKubevirt: boolean;
+  isMissingName: boolean;
+  isMissingNamespace: boolean;
+}
+
+const Spinner = () => (
+  <span className="pf-c-spinner" role="progressbar" aria-valuetext="Loading...">
+    <span className="pf-c-spinner__clipper" />
+    <span className="pf-c-spinner__lead-ball" />
+    <span className="pf-c-spinner__tail-ball" />
+  </span>
+);
+
+export const ConsoleEmptyState: React.SFC<ConsoleEmptyStateProps> = ({
+  isKubevirt,
+  isMissingName,
+  isMissingNamespace,
+}) => {
+  if (isMissingName || isMissingNamespace) {
+    return (
+      <div className="kv-cloud-vm-console-empty">
+        <EmptyState>
+          <EmptyStateIcon icon={ErrorCircleOIcon} />
+          <Title size="lg" headingLevel="h4">
+            Missing Required URL Parameters
+          </Title>
+          <EmptyStateBody>
+            <TextContent>
+              <TextList component={TextListVariants.dl}>
+                {isMissingName && (
+                  <>
+                    <TextListItem component={TextListItemVariants.dt}>name</TextListItem>
+                    <TextListItem component={TextListItemVariants.dd}>
+                      Missing name of the VM/VMI
```

Comment:
Nit: I'd omit the "Missing" prefix since it's already mentioned above ("Missing Required URL Parameters").

Reply:
```suggestion
                      Name of the VM/VMI
```